documentation/assemblies/managing/assembly-drain-cleaner.adoc

```diff
@@ -3,12 +3,12 @@
 // using/assembly-management-tasks.adoc
 
 [id='assembly-drain-cleaner-{context}']
-= Evicting pods with Strimzi Drain Cleaner
+= Evicting pods with the Strimzi Drain Cleaner
 
 [role="_abstract"]
-Kafka and ZooKeeper pods might be evicted during Kubernetes upgrades, maintenance or pod rescheduling.
+Kafka and ZooKeeper pods might be evicted during Kubernetes upgrades, maintenance, or pod rescheduling.
 If your Kafka broker and ZooKeeper pods were deployed by Strimzi, you can use the Strimzi Drain Cleaner tool to handle the pod evictions.
-Since the Strimzi Drain Cleaner will handle the eviction instead of Kubernetes, you need to set the `podDisruptionBudget` for your Kafka deployment to `0` (zero).
+Since the Strimzi Drain Cleaner handles the eviction instead of Kubernetes, you need to set the `podDisruptionBudget` for your Kafka deployment to `0` (zero).
```

Comment:
According to the IBM style guide "since" should not be used as a snynonym for "because". Please consider rewording

```suggestion
The Strimzi Drain Cleaner handles the eviction instead of Kubernetes. You must set the `podDisruptionBudget` for your Kafka deployment to `0` (zero).